src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import ball.characteristics.Ball;
+import java.util.Random;
+
+public class Lottery {
+    public static String getRandomBall() {
```

Comment:
The method name `getRandomBall` suggests that it should return a `Ball` object, but it currently returns a `String`. Consider changing the return type to `Ball` and moving the string representation to a `toString` method in the `Ball` class.